test/db/formats/dyldcache

```diff
@@ -657,11 +655,11 @@ canary   false
 PIE      false
 RELROCS  false
 NX       false
-nth paddr      vaddr       bind   type size lib name                      
---------------------------------------------------------------------------
+nth paddr      vaddr       bind   type size lib name                            
+--------------------------------------------------------------------------------
 0   0x00007fb4 0x180007fb4 GLOBAL FUNC 0        _i_am_libSystem
 0   0x0000bfb0 0x18000bfb0 GLOBAL FUNC 0        _say_hello
-0   0x0000ff38 0x18000ff38 GLOBAL FUNC 0        dyld3::entryVectorForDyld
+0   0x0000ff38 0x18000ff38 GLOBAL FUNC 0        __ZN5dyld318entryVectorForDyldE
```

Comment:
This one looks like a regression?

Reply:
It's because if fails to guess the language of the dyldcache

Reply:
It is C++, Objective-C and Swift in the same binary here. Can we handle that somehow?